packages/components/src/DateTimePickers/YearPicker/YearPicker.scss

```diff
@@ -0,0 +1,42 @@
+
+.container {
+    display: flex;
+    flex-direction: column;
+    height: 100%;
+
+    .previous, .next {
+        background: none;
+        border: none;
```

Comment:
You can use 
```
<Action
    icon={'talend-chevron'}
    className={theme.next}
    label={t('TC_YEAR_PICKER', { defaultValue: 'Show next page of years' )}
    hideLabel
    link>
```
instead to have an icon button without button style.
With that you'll use the existing component/style, adding only some small css customisation